gold-cc-cvc-input.html

```diff
@@ -173,9 +239,44 @@
         ],
 
         ready: function() {
-          // If there's an initial input, validate it.
-          if (this.value)
+          if (!this.value && Polymer.Element) {
+            this.value = '';
+          }
+
+          else if (this.value && !Polymer.Element) {
+             this._handleAutoValidate();
+          }
+        },
+
+        beforeRegister: function() {
+          this._beforeRegister();
+        },
+
+        _beforeRegister: function() {
+          var template = Polymer.DomModule.import('gold-cc-cvc-input', 'template');
+          var version = Polymer.Element ? 'v1' : 'v0';
+          var inputTemplate = Polymer.DomModule.import('gold-cc-cvc-input', 'template#' + version);
+          var inputPlaceholder = template.content.querySelector('#template-placeholder');
+          inputPlaceholder.parentNode.replaceChild(inputTemplate.content, inputPlaceholder);
```

Comment:
I think to keep this element extensible this should be wrapped in an `if (inputPlaceholder)` like in [`paper-input`](https://github.com/PolymerElements/paper-input/blob/master/paper-input.html#L267)